docs/source/examples/notebooks/rpt-experiment.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
<p>A few minor style comments:</p><ul><li>Use bullet points (<code>*</code> in markdown)</li><li>Format the variables (e.g. <code>cccv_experiment</code>) as inline code, using <code>`</code> before and after the variable.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pybamm-team/PyBaMM/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>